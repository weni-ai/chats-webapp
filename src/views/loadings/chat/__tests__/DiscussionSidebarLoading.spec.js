import { mount } from '@vue/test-utils';
import DiscussionSidebarLoading from '../DiscussionSidebar.vue';
import { beforeEach, describe, expect } from 'vitest';

const createWrapper = () => {
  return mount(DiscussionSidebarLoading);
};

describe('DiscussionSidebarLoading.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = createWrapper();
  });

  it('to match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
