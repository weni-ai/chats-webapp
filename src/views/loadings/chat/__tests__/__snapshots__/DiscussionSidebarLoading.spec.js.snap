// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DiscussionSidebarLoading.vue > to match snapshot 1`] = `
"<div data-v-89c6dd73="" class="discussion-sidebar-loading">
  <header data-v-89c6dd73="">
    <div data-v-89c6dd73=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></div>
  </header>
  <main data-v-89c6dd73=""><span data-v-89c6dd73=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-89c6dd73=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-89c6dd73=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-89c6dd73=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-89c6dd73=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-89c6dd73="" class="item-last"><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></main>
</div>"
`;
